.github/workflows/manylinux_2014.yml

```diff
@@ -0,0 +1,193 @@
+          docker run --rm \
+            -v ${{ env.OPENVINO_REPO }}:/work/src \
+            -v ov_build_cache:/work/build \
+            -v ${{ env.INSTALL_DIR }}:/work/install \
+            -e SCCACHE_AZURE_BLOB_CONTAINER \
+            -e SCCACHE_AZURE_CONNECTION_STRING \
+            -e SCCACHE_SERVER_PORT \
+            -e SCCACHE_IGNORE_SERVER_IO_ERROR \
+            -e SCCACHE_CACHE_SIZE \
+            -e SCCACHE_AZURE_KEY_PREFIX \
+            -e CMAKE_CXX_COMPILER_LAUNCHER \
+            -e CMAKE_C_COMPILER_LAUNCHER \
+            -w /work/src \
+            ${{ fromJSON(needs.docker.outputs.images).ov_build.manylinux2014_x86_64 }} \
+            /bin/bash -c "
+              cmake -DENABLE_CPPLINT=OFF -DENABLE_NCC_STYLE=OFF -DCMAKE_VERBOSE_MAKEFILE=ON -DBUILD_SHARED_LIBS=ON -DENABLE_PYTHON=OFF -DENABLE_WHEEL=OFF -S /work/src -B /work/build &&
+              cmake --build /work/build --parallel $(nproc) --config ${{ env.CMAKE_BUILD_TYPE }} &&
+              cmake --install /work/build --config ${{ env.CMAKE_BUILD_TYPE }} --prefix /work/install
+            "
+            
+      - name: Pack Artifacts
+        run: mkdir -p ${{ env.BUILD_DIR }} && tar -cvf - * | pigz > ${{ env.BUILD_DIR }}/openvino_package.tar.gz
+        working-directory: ${{ env.INSTALL_DIR }}
+            
+      - name: Build Python API(Python 3.9-3.13)
+        run: |
+          SUPPORTED_PYTHON_VERSIONS=("3.9" "3.10" "3.11" "3.12" "3.13")
+          for PY_VER in "${SUPPORTED_PYTHON_VERSIONS[@]}"; do
+            python_path=/opt/python/cp${PY_VER//./}-cp${PY_VER//./}
+            python_include=${python_path}/include/python${PY_VER}
+            python_lib=${python_path}/lib/python${PY_VER}
+            docker run --rm \
+              -v ${{ env.OPENVINO_REPO }}:/work/src \
+              -v ${{ env.INSTALL_WHEELS_DIR }}:/work/wheels \
+              -v ${{ env.PIP_CACHE_PATH }}:/work/pip_cache \
+              -v ov_build_cache:/work/build \
+              -e SCCACHE_AZURE_BLOB_CONTAINER \
+              -e SCCACHE_AZURE_CONNECTION_STRING \
+              -e SCCACHE_SERVER_PORT \
+              -e SCCACHE_IGNORE_SERVER_IO_ERROR \
+              -e SCCACHE_CACHE_SIZE \
+              -e SCCACHE_AZURE_KEY_PREFIX \
+              -e CMAKE_CXX_COMPILER_LAUNCHER \
+              -e CMAKE_C_COMPILER_LAUNCHER \
+              -w /work/src \
+              ${{ fromJSON(needs.docker.outputs.images).ov_build.manylinux2014_x86_64 }} \
+              /bin/bash -c "
+                export PATH=${python_path}/bin:\$PATH
+                PIP_VER=$(python3 -c "import pip; print(pip.__version__)")
+                export "PIP_CACHE_DIR=/work/pip_cache/${PIP_VER}"
+                python3 -m pip install -r /work/src/src/bindings/python/wheel/requirements-dev.txt &&
+                cmake -DPython3_EXECUTABLE=${python_path}/bin/python3 -DPython3_INCLUDE_DIR=${python_include} -DPython3_LIBRARY=${python_lib} -DENABLE_PYTHON=ON -DENABLE_WHEEL=ON -S /work/src -B /work/build &&
```

Comment:
I suppose `-DPython3_INCLUDE_DIR=${python_include} -DPython3_LIBRARY=${python_lib}` are not required as those are detected based on `Python3_EXECUTABLE`

why developer package is not used?
typically, we build via 
```bash
cmake -DOpenVINODeveloperPackage_DIR=/work/build -B /work/build_python_$PY_VER -S /work/src/src/bindings/python
make ie_wheel
```
This approach is currently used in Jenkins